src/store/actions/airline.js

```diff
@@ -37,32 +39,37 @@ export const fetchAirlinesStart = () => {
     }
 };
 
-export const fetchAirlines = (offset, limit) => {
+
+export const fetchAirlines = (offset, limit, airlineName, iata, icao, fleet) => {
     return dispatch => {
-        dispatch(fetchAirlinesStart());
-        // eslint-disable-next-line no-useless-concat
-        //let queryString = '?' + 'offset=' + offset + '&' + 'limit=' + limit;
-        let queryString = limit !== "-1" 
```

Comment:
This is not safe way to create query-string. Instead, use URLSearchParams class. Ex:
const queryString = new URLSearchParams();
queryString.append('offset', offset);
queryString.append('limit', limit);
queryString.append('airlineName', airlineName);
...

Reply:
Done.

Reply:
ok